todo.txt

```diff
@@ -0,0 +1 @@
+Have a wa to sing config.ts between the extension and the server.
```

Comment:
```suggestion
Have a way to sync config.ts between the extension and the server.
```
😉

Reply:
Deleted this file anyway. I must have been falling asleep as I wrote this...

Reply:
Yeah I figured this was just a placeholder while you were working anyway. 😄